go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/hashicorp/memberlist v0.2.4
 	github.com/k8snetworkplumbingwg/sriov-cni v2.1.0+incompatible
 	github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd
+	github.com/mgechev/revive v1.0.9 // indirect
```

Comment:
this change is not right and if the CI tidiness check is passing, then there is something wrong with it

Reply:
you are right, tidy check failed, I will update it.